src/Hazelcast.Net.Tests/Clustering/ConnectMembersTests.cs

```diff
@@ -73,7 +73,7 @@ await foreach (var request in memberConnectionQueue.WithCancellation(cancellatio
                 Assert.That(addresses.Count, Is.EqualTo(2));
                 Assert.That(addresses, Does.Contain(NetworkAddress.Parse("127.0.0.1:1")));
                 Assert.That(addresses, Does.Contain(NetworkAddress.Parse("127.0.0.1:2")));
-            }, 2000, 200);
+            }, 3000, 200);
```

Comment:
I think this is the timeout. This is by default 60 seconds in Node.js I guess if you don't specify it. You can do something similar I think. If there is optional parameters in C#, you can set a relatively high defualt value to avoid unstable tests.

Reply:
It shouldn't take that time actually but time to time they work slow on Github.